nanofi/tests/CSite2SiteTests.cpp

```diff
@@ -18,31 +18,59 @@
 
 #include <stdlib.h>
 
-#include <uuid/uuid.h>
 #include <algorithm>
+#include <chrono>
 #include <string>
 #include <memory>
 #include <utility>
 #include <map>
+#include <thread>
+
 #include "io/BaseStream.h"
 #include "TestBase.h"
 #include "unit/SiteToSiteHelper.h"
 #include "sitetosite/CPeer.h"
 #include "sitetosite/CRawSocketProtocol.h"
 #include "sitetosite/CSiteToSite.h"
-#include <algorithm>
-#include <core/cxxstructs.h>
+#include "sitetosite/RawSocketProtocol.h"
+#include "uuid/uuid.h"
+#include "core/cstructs.h"
+#include "RandomServerSocket.h"
+#include "core/log.h"
 
 #define FMT_DEFAULT fmt_lower
 
-TEST_CASE("TestSetPortId", "[S2S1]") {
-  auto stream_ptr = std::unique_ptr<minifi::io::BaseStream>(new org::apache::nifi::minifi::io::BaseStream());
-
-  cstream cstrm;
-  cstrm.impl = stream_ptr.get();
+const char * ATTR_NAME = "some_key";
+const char * ATTR_VALUE = "some value";
+const char * PAYLOAD = "Test MiNiFi payload";
+const char * PAYLOAD_CRC = "2006463717"; // Depends on both payload and attributes, do NOT change manually!
+std::string CODEC_NAME = "StandardFlowFileCodec";
+
+struct S2SReceivedData {
+  bool request_type_ok;
```

Comment:
This is potentially uninitialized.

Reply:
Gets assigned in both branches of a condition, but added a false here to make sure, thanks!